admin_guide/iptables.adoc

```diff
@@ -11,18 +11,89 @@
+services that are commonly used in the system that interact with the kernel iptables and can
+accidentally impact {product-title} and Docker.
+
+=== iptables
+Iptables is a low level tool that can be used to set up, maintain, and inspect the tables of IPv4
+packet filter rules in the Linux kernel.
+
+WARNING: This is a very powerful tool that can manage all aspects of the kernel iptables. It assumes
+you know what you are doing.
+
+Independent of other use, such as a firewall, {product-title} and Docker manage chains in some of the tables.
+The chains are inserted in specific order and the rules are specific to their needs.
+
+IMPORTANT: iptables --flush [chain] can remove key required configuration. Be very careful when using this.
+
+When making changes consider limiting your changes to the OS_FIREWALL_ALLOW chain in the filter table.
+
+
+=== firewalld
+
+When you need to set up a firewall on the cluster's nodes, use firewalld.
+Firewalld can be used to safely manage access to the cluster in a way that
+does not confiict with {product-title} or Docker. For example, when a
+
+----
+# firewall-cmd --reload
+----
+
+command is executed both Openshift and Docker are notified so they can restore any needed rules.
+
+Detailed instructions for working with firewalld are beyond the scope of this document.
+
+
+=== iptables.service
+The iptables service supports a network firewall. It assumes total control of the iptables configuration
+and it flushes and restores the complete iptables configuration. It is not integrated with either Docker
+or {product-title} so when it is started it breaks cluster networking.
+
+WARNING: Stopping and starting the iptables.service will destroy configuration that is required by
+{product-title} and docker. {product-title} and Docker are not notified of the change.
+
+WARNING: Use of iptables.service is not recommended in conjunction with {product-title} or Docker.
+
+Iptables.service can be disabled and masked to prevent accidental execution:
+
+----
+# systemctl disable iptables.service
+# systemctl mask iptables.service
+----
+
+If you need to run iptables.service keep a limited configuration in the config file and
+rely on docker and openshift to install their needed rules.
+
+The iptables.service configuration is loaded from:
+
+----
+/etc/sysconfig/iptables
+----
+
+To make permanent rules changes edit the changes into this file. Do not include Docker or {product-title} rules.
+
+After iptables.service is started on a node, docker must be restarted and atomic-openshift-node.service
+must be restarted to reconstruct the needed iptables configuration. Restarting docker will cause all containers
+running on the node to be stopped and restarted.
 
 ----
-$ iptables-save > /path/to/iptables.bkp
-$ systemctl restart iptables
-$ iptables-restore < /path/to/iptables.bkp
+# systemctl restart iptables.service
+# systemctl restart docker
+# systemctl restart atomic-openshift-node.service
```

Comment:
I think we also need to lay out the alternate way of saving everything to /etc/sysconfig/iptables when running, editing that, and restarting iptables.service.  That doesn't necessitate a restart of docker, but we should advise against it because it will make the riles file hideous and impossible to understand.